### PR TITLE
Recursion - binomial coefficient

### PR DESCRIPTION
**Pull request by DevratAgarwal**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [ ] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [ ] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [ ] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [ ] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Comment by vinthaivendhan10x** [comment]
@satya-9 Kindly Review

File: solutions/recursion/binomial-coefficient.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+
+
+function binomialCoeff(n, r) {
```
Comment (satya-9) [inline solutions/recursion/binomial-coefficient.js:15]: @DevratAgarwal  function name can be binomialCoefficient if it is okay

File: solutions/recursion/binomial-coefficient.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+}
+
+let testCases = parseInt(readLine());
+for (let index = 0; index < testCases; index++) {
+    let input = readLine().split(" ");
+    n = parseInt(input[0]);
+    let r = parseInt(input[1]);
+    console.log(binomialCoeff(n, r));
```
Comment (satya-9) [inline solutions/recursion/binomial-coefficient.js:34]: same here

File: solutions/recursion/binomial-coefficient.js (hunk 1 above)
Comment (DevratAgarwal) [reply solutions/recursion/binomial-coefficient.js:15]: @satya-9 that's template of the code should i change it ?

**Comment by vinthaivendhan10x** [comment]
@DevratAgarwal @rpapavan97 kindly Update

File: solutions/recursion/binomial-coefficient.js (hunk 1 above)
Comment (satya-9) [reply solutions/recursion/binomial-coefficient.js:15]: @DevratAgarwal  Template I changed. Please check once again

Comment (DevratAgarwal) [reply solutions/recursion/binomial-coefficient.js:15]: @satya-9 it is not visible

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**